lib/member_page.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :birth_date do
+    date_from(
+      line_for('Datëlindja').match(/(\d\d\.\d\d\.\d\d\d\d)/) { Regexp.last_match(1) }.to_s.tidy
```

Comment:
This part can be simplified to `line_for('Datëlindja')[/(\d\d\.\d\d\.\d\d\d\d)/, 1]`